tools/proofers/codepoint_validator.js

```diff
@@ -58,9 +62,67 @@ $(function () {
     });
 
     $("#cc-remove").click(function () {
-        let textarea = $("#text_data");
-        textarea.val(textarea.val().replace(pattern , ""));
+        textArea.value = textArea.value.replace(badPattern , "");
         $("#checker").hide();
         $("#proofdiv").show();
     });
+
+    var above = {
+        "=": "\u0304", // macron
+        ":": "\u0308", // diaeresis
+        ".": "\u0307", // dot
+        "`": "\u0300", // grave
+        "'": "\u0301", // acute
+        "^": "\u0302", // circumflex
+        ")": "\u0306", // breve
+        "~": "\u0303", // tilde
+        ",": "\u0327", // cedilla
+        "v": "\u030C", // caron
+    };
+    var below = {
+        "=": "\u0331", // macron
+        ":": "\u0324", // diaeresis
+        ".": "\u0323", // dot
+        "`": "\u0316", // grave
+        "'": "\u0317", // acute
+        "^": "\u032D", // circumflex
+        ")": "\u032E", // breve
+        "~": "\u0330", // tilde
+        ",": "\u0327", // cedilla
+        "v": "\u032C", // caron
+    };
+
+    $(textArea).on("input", function() {
```

Comment:
Please add a comment within anonymous function on what this is doing.

Reply:
Thanks for the update, but I was thinking something a little more high-level, eg:
```
Detect if the user has input one of the diacritical markups [..]
and convert it to a Unicode codepoint. If the codepoint is not
valid for the project, leave it as the markup. This also converts
a few common ligatures to their Unicode codepoints.

This event fires on every input into the textarea, but only
takes action if the last character entered is a ].
```